Makefile

```diff
@@ -1,7 +1,35 @@
 SHELL := /bin/bash -o pipefail -o errexit
 
+# You can override all of these variables on the command line like so:
+# ENV_NAME=dev TMPDIR=$HOME make test
+ENV_NAME ?= "conda-build"
+DOC_ENV_NAME ?= "conda-build-docs"
+TMPDIR ?= /tmp
+PYTHON_VERSION ?= 3.8
+
+# Setup env for documents
 env-docs:
-	conda create --name conda-build-docs --channel defaults python=3.8 --yes
-	conda run --name conda-build-docs pip install -r ./docs/requirements.txt
+	conda create --name $(DOC_ENV_NAME) --channel defaults python=$(PYTHON_VERSION) --yes
+	conda run --name $(DOC_ENV_NAME) pip install -r ./docs/requirements.txt
 
 .PHONY: $(MAKECMDGOALS)
+
+# Runs all tests
+.PHONY: test
+test: ../conda_build_test_recipe
+	conda run --live-stream -n $(ENV_NAME) pytest tests/ --basetemp $(TMPDIR)
```

Comment:
I think there is a conda run bug because without `--live-stream` you get no output at all until the end. For me I get the `Beginning tests` output at least, but then it hangs till the end again. I'll see if I can conditionally set `basetemp` based on permissions. I think other people will have the same problem.

Reply:
Ok, I made TMPDIR conditional now based on write permissions. It worked for me on my machine, but could you check it?

Reply:
Still erroring out for me, but it might be something wrong on my end; I think having someone else also test it out would be ideal!

Reply:
You _probably_ way --no-capture-output, not --live-stream.